libs/model/src/community/CreateCommunity.command.ts

```diff
@@ -0,0 +1,374 @@
+        ...new Set(
+          [...social_links, website, telegram, discord, element, github].filter(
+            (a) => a,
+          ),
+        ),
+      ];
+
+      const baseCommunity = await models.Community.findOne({ where: { base } });
+      mustExist('Chain Base', baseCommunity);
+
+      const admin_address = await findBaseAdminAddress(actor, payload);
+      if (
+        !mustExist(
+          `User address ${payload.user_address} in ${base} community`,
+          admin_address,
+        )
+      )
+        return;
+
+      // == command transaction boundary ==
+      await models.sequelize.transaction(async (transaction) => {
+        const [node] = await models.ChainNode.scope(
+          'withPrivateData',
+        ).findOrCreate({
+          where: { url: node_url },
+          defaults: {
+            url: node_url,
+            eth_chain_id,
+            cosmos_chain_id,
+            alt_wallet_url,
+            private_url,
+            balance_type:
+              base === ChainBase.CosmosSDK
+                ? BalanceType.Cosmos
+                : base === ChainBase.Substrate
+                  ? BalanceType.Substrate
+                  : base === ChainBase.Solana
+                    ? BalanceType.Solana
+                    : BalanceType.Ethereum,
+            // use first chain name as node name
+            name,
+          },
+          transaction,
+        });
+
+        await models.Community.create({
+          id,
+          name,
+          default_symbol,
+          icon_url,
+          description,
+          network,
+          type,
+          social_links: uniqueLinksArray,
+          base,
+          bech32_prefix,
+          active: true,
+          chain_node_id: node.id,
+          token_name,
+          has_chain_events_listener:
+            network === 'aave' || network === 'compound',
+          default_page: DefaultPage.Discussions,
+          has_homepage: 'true',
+          collapsed_on_homepage: false,
+          custom_stages: [],
+          directory_page_enabled: false,
+          snapshot_spaces: [],
+          stages_enabled: true,
+        });
+
+        await models.Topic.create({
+          community_id: id,
+          name: 'General',
+          featured_in_sidebar: true,
+        });
+
+        await models.Address.create({
+          user_id: actor.user.id,
+          address: admin_address.address,
+          community_id: id,
+          hex:
+            base === ChainBase.CosmosSDK
+              ? bech32ToHex(admin_address.address)
+              : undefined,
+          verification_token: admin_address.verification_token,
+          verification_token_expires: admin_address.verification_token_expires,
+          verified: admin_address.verified,
+          wallet_id: admin_address.wallet_id,
+          is_user_default: true,
+          role: 'admin',
+          last_active: new Date(),
+          ghost_address: false,
+          is_banned: false,
+        });
+      });
```

Comment:
transaction not passed to these 3 mutations

Reply:
ha.. nice catch... I was passing it by osmosis